components/sidebar.component.js

```diff
@@ -0,0 +1,18 @@
+export class SideBarComponent {
+  #page;
+  #burgerMenuBtn;
```

Comment:
cuts and abbreviations it's not a good stuff


```suggestion
  #burgerMenuButton;
```

Reply:
...hereinafter

Reply:
fixed